### DISCUSSION TITLE
Quarkus CXF v1.0 release

### DISCUSSION THREAD

**Discussion by shumonsharif** [quarkiverse/quarkus-cxf#412] [General]
Hello @quarkiverse/quarkiverse-cxf @famod ...

I believe we've been long been overdue for a v1.0 release of the extension.  

With almost 22k downloads for `io.quarkiverse.cxf` artifacts in the month of March 2022 from Maven Central, and very few new issues being opened in recent months, I believe we're in a stable and strong position to provide a GA v1.0 release.  I'm thinking of targeting around April 12th, and once Quarkus 2.8 is officially available.

Please feel free to share any thoughts.

**Comment by famod** [comment]
Hi @shumonsharif, 👍 from me!

**Reply by shumonsharif** [reply]
Release 1.0.0 is out!  What a coincidence, just noticed my 100th commit was for publishing release 1.0.0!
